lib/ejs.js

```diff
@@ -140,9 +140,24 @@ function getIncludePath(path, options) {
   var views = options.views;
   var match = /^[A-Za-z]+:\\|^\//.exec(path);
 
+  // Try to resolve on multiple directories
+  var tryPathsList = function(paths) {
```

Comment:
Is there some specific reason this needs to be defined as an internal function that relies on a closure-scoped variable? No good reason I can see for adding the overhead of creating a function object every time `getIncludePath` is called. I would pull it out, and have it return a value I can set `includePath` to.

Reply:
Any good reason, just my fault :smile: 
Agree with you, I'll commit a fix